games-emulation/jrommanager/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
```

Comment:
add gentoo-proxy-maintainers last as you are not a developer

Reply:
Do I must exactly add this?  
`<maintainer type="project">`
`    <email>proxy-maint@gentoo.org</email>  `
`    <name>Proxy Maintainers</name>  `
`</maintainer>  `

Reply:
Yes, the way you added it below is fine. However, you yourself are not a `project` but a `person`.